src/apic-extension/azext_apic_extension/command_patches.py

```diff
@@ -338,22 +338,43 @@ class CreateApimIntegration(DefaultWorkspaceParameter, CreateIntegration):
     def _build_arguments_schema(cls, *args, **kwargs):
         # pylint: disable=protected-access
         args_schema = super()._build_arguments_schema(*args, **kwargs)
+        # Remove the azure-api-management-source parameter
+        args_schema.azure_api_management_source._registered = False
+        # Remove the amazon-api-gateway-source parameter
+        args_schema.amazon_api_gateway_source._registered = False
+
+        # Create arg group for AzureApiManagementSource
+        args_schema.apim_resource_id = AAZResourceIdArg(
```

Comment:
May we don't need this parameter, as we will set the value of `azure_api_management_source` parameter in `pre_operations`?